### ISSUE TITLE
Need builds for Apple Silicon (arm) chips.

### ISSUE THREAD

**Issue by crewshin** [betaflight/betaflight-configurator#2882]
### Is your feature request related to a problem? Please describe

No "problem" other than becoming outdated fairly quickly. These machines are incredible and this is the path forward for Apple Hardware.

### Describe the solution you'd like

We need a compiled binary for Apple's new M1 chips to run natively.

### Describe alternatives you've considered

NA

**Comment by calif94577** [comment]
While not absolutely necessary as it can run on Rosetta2 I concur that this would be a nice to have as an M1 Pro chip owner.

**Comment by crewshin** [comment]
"While not absolutely necessary"... yet. Rosetta has always been a temporary transition during architecture switches. I agree with you at the moment, however, this is our chance to prepare for the cut.

**Comment by haslinghuis** [comment]
No idea if this is even runs on a M1 so please check

**Comment by haslinghuis** [comment]
@crewshin using the PR (but unable to test myself) you could test it yourself if you setup a dev environment

**Comment by crewshin** [comment]
Sorry about that. I misunderstood the context as I was on my phone. Will try and give it a shot soon.

**Comment by gandlers** [comment]
I have tried to build for ARMv8 on ManjaroARM.
Tried to cherry pick this commit into the 10.8.0 release of configurator from the main betafight git hub but not a git user so having limited success.
There are plenty or warnings for storybook addons and I have noticed that the new nwjs builds for arm have very different contents between v0.27.6' and the new'nw60-arm64_2022-01-08 version but I dont know enough about java, NWJS yarn gulp and any knowledge to work out how to fix it.
when I have run yarn gulp release --armv8 the elf file created looks like standard linux32 not arm64.
